### PR TITLE
refactor: to use debug value more

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Last nite fun.

Using `useDebugValue` can help with debugging and in the DevTools quite a bit. As a result the code also get's some leaner and more consise.

### CODE REVIEW

File: packages/react/modules/components/configure-adapter/configure-adapter.tsx [hunk 1]
```diff
@@ -64,7 +103,7 @@ const ConfigureAdapter = (props: TProps) => {
        */
       setAppliedAdapterArgs(nextAdapterArgs);
```
Comment (tdeekens) [inline packages/react/modules/components/configure-adapter/configure-adapter.tsx]: CleanShot 2020-02-25 at 09 44 32@2x
CleanShot 2020-02-25 at 09 44 24@2x

These now blow to be not callable. I don't follow why. Am I missing something? I would expect TS to infer the return type correctly. /cc @emmenko

File: packages/react/modules/components/configure-adapter/configure-adapter.tsx [hunk 2]
```diff
@@ -40,12 +40,51 @@ type TProps = {
   children?: TConfigureAdapterChildren;
 };
 
-const ConfigureAdapter = (props: TProps) => {
+const useApplieAdapterArgsState = ({
```
Comment (emmenko) [inline packages/react/modules/components/configure-adapter/configure-adapter.tsx]:
```suggestion
const useAppliedAdapterArgsState = ({
```

File: packages/react/modules/components/configure-adapter/configure-adapter.tsx [hunk 3]
```diff
@@ -64,7 +103,7 @@ const ConfigureAdapter = (props: TProps) => {
        */
       setAppliedAdapterArgs(nextAdapterArgs);
     },
-    []
```
Comment (emmenko) [inline packages/react/modules/components/configure-adapter/configure-adapter.tsx]: Any reason why this needed to change?

File: packages/react/modules/components/configure-adapter/configure-adapter.tsx (hunk 1 above)
Comment (emmenko) [reply packages/react/modules/components/configure-adapter/configure-adapter.tsx]: Can't say much like this, I would need to check out the branch

**Pull request merged**